HierarchicalDataConverter/Catalyst.HierarchicalDataConverter/LoggingHelper2.cs

```diff
@@ -19,7 +19,7 @@ public static void Debug(Guid guid, string message)
             {
                 lock (threadlock)
                 {
-                    LoggingHelper.Debug(message); // This is no longer working for some reason
+                    // LoggingHelper.Debug(message); // This is no longer working for some reason
```

Comment:
It would not build with this uncommented, so I commented it out.

Reply:
Yeah - this is a recent change that is only in 18.3.  I'll be looking into logging hopefully tomorrow.